src/SldStyleParser.ts

```diff
@@ -1317,11 +1317,12 @@ export class SldStyleParser implements StyleParser {
     };
 
     const cssParameters: any[] = Object.keys(lineSymbolizer)
-      .filter((property: any) => property !== 'kind' && propertyMap[property])
+      .filter((property: any) => property !== 'kind' && propertyMap[property] &&
+        lineSymbolizer[property] !== undefined && lineSymbolizer[property] !== null)
```

Comment:
`lineSymbolizer[property] !== undefined && lineSymbolizer[property] !== null` could be simplified to `lineSymbolizer[property]` or `!!lineSymbolizer[property]`

Reply:
That would filter out properties with a value of `0` as well, no?

Reply:
My bad.